### ISSUE TITLE
New rule: no setter actions 

### ISSUE THREAD

**Issue by timdeschryver** [timdeschryver/eslint-plugin-ngrx#39]
We encourage to use actions as "events" and not as setters.
This is also something that is included in the [redux style guide](https://redux.js.org/style-guide/style-guide#model-actions-as-events-not-setters)

This rule could check if:

- an action starts with `set`, e.g. `setCustomer`, `setAmount`
- an action type starts with `set`, e.g. `[CUSTOMERS] Set Customer`, `[CHECKOUT] Set Amount`

**Comment by rafaelss95** [comment]
Instead of creating a new rule, would it be weird to implements this directly in `action-hygine` rule?

**Comment by timdeschryver** [comment]
@rafaelss95 not really, since this will also verify action method names, not only the type of an action.
